mathics/builtin/numbers/calculus.py

```diff
@@ -1633,15 +1679,61 @@ class FindMinimum(_BaseFinder):
     r"""
     <dl>
     <dt>'FindMinimum[$f$, {$x$, $x0$}]'
-        <dd>searches for a numerical root of $f$, starting from '$x$=$x0$'.
-    <dt>'FindMinimum[$lhs$ == $rhs$, {$x$, $x0$}]'
```

Comment:
Why remove this?

Reply:
An equation can have an infinite number of solutions, and Wolfram Language allows equations (read ["Details and Options"](https://reference.wolfram.com/language/ref/FindMinimum.html)).

Reply:
But this is to set constraints in the optimization, something that is not implemented yet...